libiocage/cli/start.py

```diff
@@ -35,23 +35,32 @@
 @click.option("--rc", default=False, is_flag=True,
               help="Will start all jails with boot=on, in the specified"
                    " order with smaller value for priority starting first.")
-@click.option("--log-level", "-d", default=None)
 @click.argument("jails", nargs=-1)
-def cli(ctx, rc, jails, log_level):
+def cli(ctx, rc, jails):
     """
     Starts Jails
     """
 
     logger = ctx.parent.logger
-    logger.print_level = log_level
-    ioc_jails = libiocage.lib.Jails.Jails(logger=logger)
+    ioc_jails = libiocage.lib.Jails.JailsGenerator(
+        logger=logger,
+        filters=jails
+    )
 
-    for jail in ioc_jails.list(filters=jails):
+    failed_jails = []
+    for jail in ioc_jails:
         logger.log(f"Starting {jail.humanreadable_name}")
         try:
-            jail.start()
+            ctx.parent.print_events(jail.start())
+
         except Exception:
+            failed_jails.append(jail)
+            continue
             exit(1)
 
         logger.log(f"{jail.humanreadable_name} running as JID {jail.jid}")
-        exit(0)
+
+    if len(failed_jails) > 0:
```

Comment:
I think these two can be a one liner

Reply:
Example pls :)

Reply:
```
exit(1) if len(failed_jails) > 0 else exit(0)
```

Reply:
Got your point now!! That's even a bit shorter:

```python
exit(1*bool(len(failed_jails)))
```

Reply:
Readability counts in python though ;) It's not always about shortest in that sense.